<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>206</width>
      <height>280</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>–ß–∞—Å—Ç–æ—Ç–∞ X (a)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="freq_x_lineedit">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>–ß–∞—Å—Ç–æ—Ç–∞ Y (b)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="freq_y_lineedit">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>–°–¥–≤–∏–≥ —Ñ–∞–∑ (Œ¥)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="phase_shift_lineedit">
       <property name="text">
        <string>0.25</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>–ö–æ–ª–∏—á–µ—Å—Ç–≤–æ —Ç–æ—á–µ–∫</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="resolution_lineedit">
       <property name="text">
        <string>200</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>–¶–≤–µ—Ç –ª–∏–Ω–∏–∏</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="color_combobox">
       <item>
        <property name="text">
         <string>–°–∏–Ω–∏–π</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>–ó–µ–ª—ë–Ω—ã–π</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>–ö—Ä–∞—Å–Ω—ã–π</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>–ñ—ë–ª—Ç—ã–π</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>–¢–æ–ª—â–∏–Ω–∞ –ª–∏–Ω–∏–∏</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="width_combobox">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QPushButton" name="plot_button">
       <property name="text">
        <string>–û–±–Ω–æ–≤–∏—Ç—å —Ñ–∏–≥—É—Ä—É</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>–°–æ—Ö—Ä–∞–Ω–∏—Ç—å —Ñ–∏–≥—É—Ä—É –≤ —Ñ–∞–π–ª</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_equations">
       <property name="text">
        <string>X = sin(at + Œ¥ùúã), Y = sin(bt)</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>–†–∞–∑–º–µ—Ä –≤ –ø–∏–∫—Å–µ–ª—è—Ö</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="image_size_lineedit">
       <property name="text">
        <string>400</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
